content/basics/ably.textile

```diff
@@ -0,0 +1,89 @@
+
+Alternatively take a look at our "Use cases and solutions":https://ably.com/solutions page for specific example use cases.
+
+h2(#about). About the APIs
+
+Ably supports both realtime and REST interfaces. The "Realtime library":/basics/use-ably#realtime-interface establishes a WebSocket connection allowing realtime messages to be efficently sent both to and from the Ably. The "REST library":/basics/use-ably#rest-interface communicates using the HTTP protocol for all operations and is effectively stateless.
+
+There are a number of "Ably SDKs":https://ably.com/download in different languages to match your needs.
+
+Some of the key features are: 
+
+* "PubSub with channels":#pubsub
+* "Presence and occupancy":#presence-and-occupancy
+* "Message storage and history":#message-storage-and-history
+* "Push Notifications":#push-notifications
+
+h3(#pubsub). PubSub with Channels
+
+Ably uses the Publish/Subscribe (Pub/Sub) architectural pattern to allow any number of publishers to publish data and any number of subscribers to subscribe to a source of data. This works hand in hand with Ably's concept of "channels". Channels are used to aggregate all data into a single location, allowing publishers to publish to just one location and subscribers to subscribe to just one location.
+
+Read more in our "Pub/Sub section":/channels/.
+
+h3(#presence-and-occupancy). Presence and Occupancy
+
+Ably’s presence feature allows clients or devices to announce their presence on a channel. Other devices or services may then subscribe to these presence events (such as entering, updating their state, or leaving the channel). You can also request a list of clients or devices that are online/offline on a channel at a particular point in time via the REST API.
+
+Occupancy provides metrics about the clients attached to a channel, including a count of the number of clients attached, the number of publishers and subscribers, the number of presence members, and the number of presence publishers and subscribers.
+
+Read more in our "Presence and occupancy section":/presence-occupancy/.
+
+h3(#message-storage-and-history). Message storage and History
+
+Ably's Message storage and History allows messages to be accessed after initially being sent. You can persist the last message sent using the rewind channel option or enable persistence of all messages for 24 hours for the free tier or 72 hours for paid tiers. These messages can be retrieved using the History API. You can also persist messages to external storage services such as AWS through the use of "Ably Integrations":/integrations/.
+
+By default messages are only stored for two minutes on Ably’s servers, these messages can be retrieved in the event of a lost connection using the History API.
```

Comment:
> By default messages are only stored for two minutes on Ably’s servers, these messages can be retrieved in the event of a lost connection using the History API.

Should the default situation come first, rather than last? 

Also, I'm not sure this statement is true (need to check). I think if disconnection is less than this default of two minutes, the message handling is done automatically by the SDK, and you don't need to invoke the History API to retrieve a message. However, if you've enabled message persistence and the disconnection is longer than two minutes you can use the History API to retrieve "lost" messages. (I might be wrong about this, so please check thoroughly).